dotCMS/src/main/resources/apps/dotAnalytics-config.yml

```diff
@@ -0,0 +1,48 @@
+name: "Analytics Integration"
+iconUrl: "https://static.dotcms.com/assets/icons/apps/saml.png"
+allowExtraParameters: true
+description: "Analytics integration provides the ability to, given a Client Id and Client Secret, resolve an access token to be injected across all the analytics services included the resolving of the Analytics Key (Id)."
+
+params:
+  clientId:
+    value: true
+    hidden: false
+    type: "STRING"
+    label: "Analytics Client ID"
+    hint: "Client ID is required to resolve access token from IDP"
+    required: true
+  clientSecret:
+    value: true
+    hidden: true
+    type: "STRING"
+    label: "Analytics Client Secret"
+    hint: "Client Secret is required to resolve access token from IDP"
+    required: true
+  analyticsKey:
+    value: true
+    hidden: false
```

Comment:
hidden?

Reply:
Not really. This is just to segment data to identify when capturing events. This will be visible to anyone who sees the traffic.

Reply:
It is not critical data.